README.md

```diff
@@ -102,8 +102,9 @@ Perform the following commands on the terminal and write them down for review,
   - Checkout new branch
   - Merge branch into another
   - See commit history
-  - Rebase and Squash
   - Write a command to clone [this](https://github.com/canopas/web-developer-roadmap-2023) repo
+  - Create two branches, `develop` and `feature`. Add 5 commits to `feature` and 1 commit to `develop`. Merge `feature` into `develop` using sqaush. `develop` branch should have `2 commits`, 1 previous and combined 1 from `feature`. Both branch should have same changes. 
```

Comment:
Can you convert `1 previous and combined 1 from` to `1 previous and 1 combined from`? I think this provides more clarity and readability.